snapcraft/__init__.py

```diff
@@ -446,3 +446,8 @@ def str_presenter(dumper, data):
 yaml.add_representer(str, str_presenter)
 yaml.add_representer(OrderedDict, dict_representer)
 yaml.add_constructor(_mapping_tag, dict_constructor)
+
+if common.is_snap():
```

Comment:
`common.internal` holds all the good bits and we don't want to expose `is_snap` as a public API.